api/fossa/tar.go

```diff
@@ -439,7 +440,7 @@ func tarballUpload(name string, dependency, rawLicenseScan, upload bool, tarball
 	// Queue the component build.
 	build := ComponentSpec{
 		Archives: []Component{
-			Component{PackageSpec: name, Revision: revision},
+			{PackageSpec: name, Revision: revision},
```

Comment:
Why this remove the name here? I would prefer that struct values in slices are named, to reduce ambiguity. Is there an argument for not naming them?

Reply:
I changed this because I got a linting warning and didn't have a preference either way, but if you see a benefit in keeping it I'm happy to do that. I believe the linter warns because the explicit `Component` is redundant as we are already defining the slice as `[]Component`.